homeassistant/components/soundtouch/config_flow.py

```diff
@@ -94,10 +91,10 @@ async def _async_get_device_id(self, raise_on_progress: bool = True) -> None:
 
         self.name = device.config.name
 
-    async def _async_create_soundtouch_entry(self):
+    async def _async_create_soundtouch_entry(self) -> ConfigFlowResult:
         """Finish config flow and create a SoundTouch config entry."""
         return self.async_create_entry(
-            title=self.name,
+            title=self.name or "SoundTouch",
```

Comment:
I'm not sure if the name can ever really be `None`, but the library types it as such and there are internal checks for it.
```python
        if self.name:
            # If we have a name, use it as flow title
            self.context["title_placeholders"] = {"name": self.name}
```